plugin-server/src/cdp/services/segment-destination-executor.service.ts

```diff
@@ -307,17 +306,9 @@ export class SegmentDestinationExecutorService {
                     result.invocation.queuePriority = metadata.tries
                     result.invocation.queueScheduledAt = getNextRetryTime(this.serverConfig, metadata.tries)
                     return result
-                } else {
-                    result.finished = true
```

Comment:
Why remove this line? Won't this job be picked up again?

Reply:
The default is finished true